src/state/zustand/swapperStore/amountSelectors.ts

```diff
@@ -210,21 +211,21 @@ export const selectSellAmountBeforeFeesFiat = createSelector(
 )
 
 export const selectTradeOrQuoteSellAmountBeforeFeesCryptoBaseUnit = createSelector(
-  (state: SwapperState) =>
-    state.activeSwapperWithMetadata?.quote?.sellAmountBeforeFeesCryptoBaseUnit,
+  selectActiveSwapperWithMetadata,
   (state: SwapperState) => state.trade?.sellAmountBeforeFeesCryptoBaseUnit,
-  (quoteSellAmountBeforeFeesBaseUnit, tradeSellAmountBeforeFeesBaseUnit): string | undefined =>
+  (activeSwapperWithMetadata, tradeSellAmountBeforeFeesBaseUnit): string | undefined =>
     // Use the trade amount if we have it, otherwise use the quote amount
-    tradeSellAmountBeforeFeesBaseUnit ?? quoteSellAmountBeforeFeesBaseUnit,
+    tradeSellAmountBeforeFeesBaseUnit ??
+    activeSwapperWithMetadata?.quote?.sellAmountBeforeFeesCryptoBaseUnit,
```

Comment:
What was the rationale for making this more verbose?

Reply:
the verbosity wasnt intentional, its more that `activeSwapperWithMetadata` no longer exists on the store and we need to use `selectActiveSwapperWithMetadata` and then pull out the `sellAmountBeforeFeesCryptoBaseUnit` here instead of in the first arg of this selector